### DISCUSSION TITLE
Hvordan afslår jeg arrangører?

### DISCUSSION THREAD

**Discussion by MartinQwist** [xooom-community/public#14] [Questions] [answered]
Vi kunne i KF godt bruge, at man kan afvise en arrangør. Vi har ret mange pending, som egentlig bare skal afvises. 

Kan vi få Afvis-knappen til at virke?

![image](https://github.com/user-attachments/assets/8eb09fb7-7b29-4b40-9d80-f0e95139cfe6)

**Comment by henrikbak** [comment, marked as answer]
Det kunne se ud til at afvis arrangør ikke er aktiveret i øjeblikket. Det skal vi selvfølgelig have på. Jeg har oprettet en opgave på det: https://github.com/orgs/xooom-community/projects/1/views/1?pane=issue&itemId=80212719
